specification/network/resource-manager/Microsoft.Network/stable/2021-08-01/firewallPolicy.json

```diff
@@ -1831,7 +1866,11 @@
           "enum": [
             "Ascending",
             "Descending"
-          ]
+          ],
+          "x-ms-enum": {
+            "name": "Order",
```

Comment:
in the SDK, the enum name like 'Order', 'Mode' is easy to conflict with other name or keywords.
Could you pick a better name like  FirewallPolicyResultOrder etc.

Reply:
Thank you, didn't realize this was how it worked. Will add to internal team documentation

Reply:
@lirenhe Updated to FirewallPolicyIDPSQueryOrder